app/routes/authentication.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (router) {
   router.get('/logout', function (req, res) {
     log.info({ user: req.user }, 'logout')
     req.logout()
-    return res.redirect(`${config.TOKEN_HOST}${config.LOGOUT_PATH}`)
+    return res.redirect(`${config.TOKEN_HOST}${config.LOGOUT_PATH}?redirect_uri=${config.POST_LOGOUT_URL}`)
```

Comment:
will also need the `client_id=${config.CLIENT_ID}` adding here

Reply:
Done 👍